packages/next/server/config-shared.ts

```diff
@@ -1,6 +1,7 @@
 import os from 'os'
 import { Header, Redirect, Rewrite } from '../lib/load-custom-routes'
 import { ImageConfig, imageConfigDefault } from './image-config'
+import type webpack from 'webpack'
```

Comment:
This type will be incorrect, `config` should likely be `unknown` or `any` in this particular case as webpack is compiled and it can be either webpack 4 or 5, the types referred here are webpack 4.

Reply:
ah, i had assumed it would pick up [these types](https://github.com/vercel/next.js/blob/27ccd3cb38e39a1dbfa0d9b8935063814c1f9803/packages/next/package.json#L186). i'll change to `any` then.